docs/development_suite/api-console/api-design/merge_collaboration.md

```diff
@@ -5,12 +5,13 @@ sidebar_label: Merge of configurations
 ---
 
 :::caution
-Currently, in the merge process, the following Design sections are not supported:
+During the merge process, please remember that:
 
-- Visualize
-- Advanced
+- the configuration included in the _Visualize_ are not included and any updated configuration must be manually updated after the merge
+- the configuration included in the _Advanced_ section can be merged if the [Enhanced Project Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) is enabled, otherwise its content must be updated after the merge
```

Comment:
```suggestion
- the configuration included in the _Advanced_ section can be merged if the [Enhanced Project Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) is enabled, otherwise its content must be manually updated after the merge
```